src/components/RatingWidget/Stars/Stars.tsx

```diff
@@ -1,12 +1,12 @@
 export const ActiveStar = () => {
   return (
-    <svg xmlns="http://www.w3.org/2000/svg" width="22" height="21" fill="none"><path fill="#FFBF3D" stroke="#FFBF3D" strokeWidth="1.518" d="M7.457 7.3H1.902c-.133 0-.26.039-.363.108a.558.558 0 0 0-.214.264.494.494 0 0 0-.014.31c.029.103.091.2.186.276v.001l4.727 3.748a1.176 1.176 0 0 1 .386 1.318l-1.986 5.694a.494.494 0 0 0-.002.321.553.553 0 0 0 .21.277.652.652 0 0 0 .374.114.652.652 0 0 0 .372-.115l4.876-3.418a1.229 1.229 0 0 1 1.41 0l4.874 3.417h.001a.651.651 0 0 0 .373.117.651.651 0 0 0 .373-.114.553.553 0 0 0 .212-.277.495.495 0 0 0-.002-.322l-1.987-5.698L7.457 7.3Zm0 0c.243 0 .483-.073.688-.211.204-.14.365-.339.452-.577l1.99-5.391a.56.56 0 0 1 .212-.255.653.653 0 0 1 .36-.107c.132 0 .258.039.36.107a.56.56 0 0 1 .213.255l1.99 5.39v.001c.088.238.248.438.453.577.205.139.446.21.69.21h5.55c.132.001.258.04.36.109a.563.563 0 0 1 .216.262c.038.1.043.207.016.309a.542.542 0 0 1-.18.274l-4.732 3.75c-.19.15-.332.356-.403.593L7.457 7.3Z"/></svg>
+      <svg width="24" height="20" viewBox="0 0 24 20" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M21.6955 7.21794C21.6038 6.97371 21.4391 6.76302 21.2235 6.614C21.0078 6.46498 20.7514 6.38473 20.4886 6.38396H15.2635C15.1746 6.38402 15.0879 6.35671 15.0153 6.30579C14.9427 6.25487 14.8879 6.18287 14.8583 6.09972L12.9827 0.829446C12.8909 0.585856 12.7262 0.375895 12.5106 0.227663C12.2951 0.0794312 12.039 0 11.7767 0C11.5144 0 11.2583 0.0794312 11.0428 0.227663C10.8272 0.375895 10.6625 0.585856 10.5706 0.829446L8.69505 6.09972C8.66558 6.18273 8.61087 6.25464 8.53846 6.30554C8.46606 6.35645 8.37951 6.38384 8.29074 6.38396H3.06396C2.79941 6.38376 2.54123 6.46435 2.32452 6.61476C2.10781 6.76518 1.94309 6.97812 1.85277 7.22462C1.76244 7.47113 1.7509 7.73923 1.8197 7.99247C1.88851 8.24571 2.03433 8.47179 2.23732 8.63997L6.68725 12.2993C6.75382 12.3541 6.80214 12.4275 6.82588 12.5099C6.84962 12.5924 6.84768 12.68 6.82031 12.7614L4.95072 18.3193C4.86431 18.5763 4.86261 18.8538 4.94584 19.1118C5.02908 19.3698 5.19294 19.5949 5.41373 19.7545C5.63452 19.914 5.90079 19.9998 6.17402 19.9995C6.44725 19.9991 6.71327 19.9126 6.93361 19.7524L11.5226 16.4165C11.5962 16.363 11.685 16.3342 11.7763 16.3342C11.8675 16.3342 11.9563 16.363 12.0299 16.4165L16.6172 19.7515C16.8375 19.9123 17.1036 19.9993 17.3771 20C17.6506 20.0007 17.9172 19.9151 18.1383 19.7555C18.3594 19.596 18.5235 19.3707 18.6068 19.1125C18.6902 18.8543 18.6884 18.5765 18.6018 18.3193L16.7322 12.758C16.7049 12.6766 16.7029 12.589 16.7267 12.5065C16.7504 12.4241 16.7987 12.3506 16.8653 12.2959L21.3238 8.63146C21.5242 8.46186 21.6673 8.23539 21.7337 7.98262C21.8002 7.72985 21.7869 7.46295 21.6955 7.21794Z" fill="#FFBF3D"/></svg>
```

Comment:
Can you run the SVG's through this please to reduce the size a bit. Thanks :pray: 
https://jakearchibald.github.io/svgomg/